.github/workflows/build-pr-check.yml

```diff
@@ -50,5 +54,5 @@ jobs:
       env:
         DEVELOCITY_ACCESS_KEY: ${{ secrets.DEVELOCITY_API_TOKEN }}
     - name: Build and push images
-      if: github.event_name == 'pull_request'
+      if: ${{ github.event_name == 'pull_request' && ! github.event.pull_request.head.repo.fork }}
```

Comment:
@vinokurig  maybe we should check build image and skip only push operation?

Reply:
Pushing images separately is not supported by the `build.sh` script. We can only build, or build and push.